frontend/public/locales/en/translation.json

```diff
@@ -563,13 +568,8 @@
     "creation.app.section.prePost": "Configure automation for prehook and posthook",
     "creation.app.section.settings": "Settings: Specify application behavior",
     "creation.app.section.techPreview": "Technology Preview",
-    "creation.app.settings.clusterSelector": "Deploy application resources on clusters with all specified labels",
-    "creation.app.local.clusters.only": "{{0}} deploys only to local cluster",
-    "creation.app.clusters.matching": "{{0}} deploys to clusters with these label: {{1}}",
-    "creation.app.clusters.expressions": "{{0}} deploys to clusters with these expressions: {{1}}",
-    "creation.app.clusters.all.online": "{{0}} deploys to all online clusters and local cluster",
-    "creation.app.clusters.only.online": "{{0}} deploys to all online clusters",
     "creation.app.select.no.existing.placement.rule": "The selected namespace does not contain any existing placement configurations",
+    "creation.app.settings.clusterSelector": "Deploy application resources on clusters with all specified labels",
```

Comment:
Not sure why they are changed, but seems like they are just changed in sequence

Reply:
I think John made some changes in that area recently. Maybe the sequence got changed when he delivered the new code.

Reply:
Ah I see, thanks Feng!